code_curation2_submissions/Santiage_rs_seed_repo_generator.m

```diff
@@ -1,54 +1,36 @@
 function [Result] = rs_seed_repo_generator(Seeds,BOG_rsMRI,PSN_rsMRI,MainDataFolder,PipelineName)
 % function [ADataDir,AMaskFilename, AROIDef,AResultFilename, ACovariablesDef, VorR_tag, MainOutputFolder] = seed_repo_generator(Seeds,BOG_rsMRI,PSN_rsMRI,MainDataFolder,PipelineName)
-
```

Comment:
Don't use space in between the header text. 
Once space is used, the rest of the header text will not be printed when type 'help function_name;'